packages/google_maps_flutter/google_maps_flutter/lib/google_maps_flutter.dart

```diff
@@ -24,6 +24,7 @@ export 'package:google_maps_flutter_platform_interface/google_maps_flutter_platf
         CameraPositionCallback,
         CameraTargetBounds,
         CameraUpdate,
+        CameraUpdateAnimationConfiguration,
```

Comment:
We generally don't want to expose these parameter objects at the app-facing package API level; they solve a problem (adding optional parameter being a breaking change for subclasses) that doesn't exist at this level, and add complexity for all clients (vs. just ourselves in implementation packages). The client-facing API should expose the duration directly, and that should be packed into the configuration object as an internal implementation detail.